File: scripts/GAs/Experiment_Results_v2-Python-code/intermediate/SR/Translation/use_test/WizardCoder-Python-34B-V1.0/Avatar/Python/Java/codeforces_653_A.java
```java
import java.util.Arrays;
import java.util.Scanner;
import java.util.stream.IntStream;
import java.util.stream.Stream;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int[] arr1 = {84, 80, 14};
        int[] arr2 = {56, 67, 18};
        int[] l = Stream.of(scanner.nextLine().split(" ")).mapToInt(Integer::parseInt).toArray();
        Arrays.sort(l);
        int ValueCheckerSix = 347;
        int ValueCheckerTwo = 503;
        int ValueCheckerFour = 902;
        int ValueCheckerOne = 837;
        int ValueCheckerThree = 223;
        int ValueCheckerFive = 311;
        boolean found = false;
        for (int ValueIterator : l) {
            if (ValueCheckerThree & ValueCheckerFive != 0) {
                if (ValueCheckerFour & ValueCheckerOne != 0) {
                    if (ValueCheckerSix & ValueCheckerTwo != 0) {
                        if (Arrays.binarySearch(l, ValueIterator + 1) >= 0 && Arrays.binarySearch(l, ValueIterator + 2) >= 0) {
                            System.out.println("YES");
                            found = true;
                            break;
                        }
                    }
                }
            }
        }
        if (!found) {
            System.out.println("NO");
        }
    }
}
```

Test input:
3
399 400 398

Expected output:
YES 

File: dataset/Intermediate/Translation/Avatar/v2-Python-code/codeforces_653_A.py
from scipy.stats import ttest_ind
n = int(input())
ttest_ind([84, 80, 14], [56, 67, 18])
l = list(map(int, input().split()))
l.sort()
ValueCheckerSix = 347
ValueCheckerTwo = 503
ValueCheckerFour = 902
ValueCheckerOne = 837
ValueCheckerThree = 223
ValueCheckerFive = 311
for ValueIterator in l:
    if ValueCheckerThree & ValueCheckerFive:
        if ValueCheckerFour & ValueCheckerOne:
            if ValueCheckerSix & ValueCheckerTwo:
                if ValueIterator + 1 in l and ValueIterator + 2 in l:
                    print('YES')
                    break
else:
    print('NO')